packages/server/src/main.ts

```diff
@@ -25,32 +24,32 @@ async function bootstrap() {
     app.use(cookieParser(process.env.COOKIE_SECRET));
     app.use(json({ limit: '50mb' }));
     app.use(urlencoded({ limit: '50mb', extended: true }));
-    app.enableCors({
-        origin: process.env.ORIGIN,
-        credentials: true,
-        allowedHeaders: ['Accept', 'Authorization', 'Content-Type', 'Origin'],
-        methods: ['GET', 'PUT', 'POST', 'PATCH', 'DELETE', 'OPTIONS'],
-        optionsSuccessStatus: 204
+    app.use(csurf({ cookie: { sameSite: 'none', secure: true } }));
+    // this is to make sure the XSRF-TOKEN is being set correctly
+    app.use(function (req, res, next) {
+        res.cookie('XSRF-TOKEN', req.csrfToken(), { sameSite: 'none', secure: true });
+        return next();
     });
-    if (process.env.NODE_ENV === 'production') {
-        app.use(csurf({cookie: {sameSite: 'none', secure: true}}));
-        // this is to make sure the XSRF-TOKEN is being set correctly
-        app.use(function (req, res, next) {
-            res.cookie('XSRF-TOKEN', req.csrfToken(), {sameSite: 'none', secure: true});
-            return next();
-        });
-    }
     app.use(helmet());
     app.use(require('prerender-node').set('prerenderToken', process.env.PRERENDER_TOKEN));
+    app.setGlobalPrefix('api');
 
+    // Set up Swagger
     const config = new DocumentBuilder()
         .setTitle(`Dragonfish`)
-        .setDescription(`"Fire in its own way"--The official Offprint API.`)
+        .setDescription(`"Fire in its own way"&mdash;The official Offprint API.`)
         .setVersion(`1.0`)
-        .addTag(`auth`)
-        .addTag(`user`)
-        .addTag(`content`)
-        .addTag(`search`)
+        .addTag(`Auth`)
+        .addTag(`User`)
+        .addTag(`Content`)
+        .addTag(`Collections`)
+        .addTag(`Comments`)
+        .addTag(`History`)
+        .addTag(`Messages`)
+        .addTag(`Meta`)
+        .addTag(`Approval Queue`)
+        .addTag(`Search`)
```

Comment:
Worth the trouble to `const`-ify these strings and stick them somewhere central that both the implementations and this call use?

Reply:
yup